governance/xc_admin/packages/xc_admin_frontend/pages/_app.tsx

```diff
@@ -6,12 +6,9 @@ import {
 import { WalletModalProvider } from '@solana/wallet-adapter-react-ui'
 import '@solana/wallet-adapter-react-ui/styles.css'
 import {
-  GlowWalletAdapter,
   LedgerWalletAdapter,
   PhantomWalletAdapter,
   SolflareWalletAdapter,
-  SolletExtensionWalletAdapter,
-  SolletWalletAdapter,
```

Comment:
fyi @guibescos these wallet adapters have been deprecated and no longer exist in the current version of the solana wallet adapter library.